Client/Application/AppDelegate.swift

```diff
@@ -341,7 +330,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
             quickActions.launchedShortcutItem = nil
         }
 
-        UnifiedTelemetry.recordEvent(category: .action, method: .foreground, object: .app)
+        TelemetryWrapper.recordEvent(category: .action, method: .foreground, object: .app)
```

Comment:
Is this a duplicate to what Glean does internally already?

Reply:
Not really, there isn't a generic `recordEvent` function in Glean like there is in legacy telemetry. Each event is treated as an object to call `record` on.  If you look at the rather large switch statement, you can see how each of the events are called based on the category/method/object/value.

Reply:
The wrapper here keeps all of that in one place, but I'm not sure if there are any performance hits due to the size of the resulting switch statement.

Reply:
Sorry, I meant the `foreground/background` stuff, I thought glean was already doing something in that regard

Reply:
Oh, yes, Glean captures the foreground/background events through the baseline pings.

Reply:
In this case, the event is being ignored and should fall into the `default:` case of the switch.